### PR TITLE
Bug fix for Barcodes groups taking longer to respond

### PR DESCRIPTION
**Pull request by Sangeetha-Bheeman**
Closes #

Changes proposed in this pull request:

*
*
* ...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,6 +57,8 @@ The following tools are required for development:
         brew install postgresql@9.6
         brew link postgresql@9.6 --force
 
+      postgresql@9.6 is used in prod, since it is not supported any more, installing the latest version for development should work.
```
Comment (sdjmchattie) [inline README.md:60]: If it's no longer supported, should we be making plans to upgrade it in production?  Definitely worth bringing up with the rest of the team.

File: migrations/versions/bcd74f0a02ea_added_index_for_barcodes_group_id_in_.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+Revision ID: bcd74f0a02ea
+Revises: e505f2e15499
+Create Date: 2022-10-11 15:23:43.608690
+
+"""
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "bcd74f0a02ea"
+down_revision = "e505f2e15499"
```
Comment (sdjmchattie) [inline migrations/versions/bcd74f0a02ea_added_index_for_barcodes_group_id_in_.py:12]: Is this right?  I don't see an `e505f2e15499` revision in the repository.

**Review by sdjmchattie — CHANGES_REQUESTED** [review]
Generally looks good.  Definitely need to make sure we get the revision numbers correct.  Plus we really should discuss the fact that we are using an unsupported database version in production.

File: README.md (hunk 1 above)
Comment (Sangeetha-Bheeman) [reply README.md:60]: Yes, I was discussing this with @emrojo. We need to inform the DBA about it I guess.

File: migrations/versions/bcd74f0a02ea_added_index_for_barcodes_group_id_in_.py (hunk 2 above)
Comment (Sangeetha-Bheeman) [reply migrations/versions/bcd74f0a02ea_added_index_for_barcodes_group_id_in_.py:12]: I think the down_revision - "e505f2e15499" was referring to this migration -migrations/versions/e505f2e15499_add_child_barcode_counter_table.py

Comment (sdjmchattie) [reply migrations/versions/bcd74f0a02ea_added_index_for_barcodes_group_id_in_.py:12]: Wow yes sorry.  I thought my local copy was up to date, but it was quite old.  I see that now.

**Review by sdjmchattie — APPROVED** [review]

**Pull request merged**